Makefile

```diff
@@ -155,6 +155,18 @@ navigation-fixtures:
 	./gradlew :libtrip-notification:artifactoryPublish
 	./gradlew :libnavigation-core:artifactoryPublish
 
+.PHONY: 1.0-core-dependency-graph
+1.0-core-dependency-graph:
+    ./gradlew :libnavigator:generateDependencyGraphMapboxLibraries
```

Comment:
Same as above re: `Makefile` formatting (if it works for you locally scratch this comment).